src/WidgetBase.ts

```diff
@@ -198,13 +191,14 @@ export class WidgetBase<P = WidgetProperties, C extends DNode = DNode> extends E
 		this._properties = <P> {};
```

Comment:
The honest answer is I don't know how significant it would be, I do suspect minimal.